gc_registry/certificate/routes.py

```diff
@@ -90,37 +93,42 @@ def create_issuance_metadata(
 
 @router.post(
     "/transfer",
-    response_model=GranularCertificateActionRead,
```

Comment:
Should maybe consider bespoke response models?

Reply:
Yeah maybe, on the fence about what to actually return for actions, as in for queries it's obvious to just return the bundles but for e.g. cancellation, maybe just a confirmation of the action? Because they will have selected those IDs to cancel in a query beforehand. If it's just a confirmation then that can be generic I would have thought